src/code-editor/interfaces.ts

```diff
@@ -148,6 +148,7 @@ export namespace CodeEditorProps {
 
     editorGroupAriaLabel?: string;
     statusBarGroupAriaLabel?: string;
+    cursorPositionAriaLabel?: (row: number) => string;
```

Comment:
I think this is an opportunity to streamline the docs for the CodeEditors `i18n` property. Currently the API docs for the property contains just a few of them. I think we could add an explanaition for all properties including an example similar to what we do at the [AppLayout's API doc](https://cloudscape.design/components/app-layout/?tabId=api#:~:text=false-,ariaLabels,-AppLayoutProps.Labels).

Let me know what you think about.